packages/cli/src/index.ts

```diff
@@ -125,6 +126,11 @@ program
     (value) => localeValidator(value, '-l'),
     'en'
   )
+  .option(
+    '-b, --button-selectors <path>',
```

Comment:
Would `--gdpr-config` make it more clear (and the shortcut `-g`) ?

Reply:
Need Alberto's confirmation on this one. I have added this as is for now.